bin/likely_name_split.py

```diff
@@ -57,8 +57,9 @@ def count_names(self, anthology):
     # returns ("Lopez Gonzalez", "Maria Victoria")
     # uses counts of words in first and last names in current database
     def best_split(self, name):
-        if "," in name:
+        if "," in name and not "Jr." in name:
             # Short-circuit names that are already split
```

Comment:
Should we handle other exceptions, too, or just deal with them as they come? e.g., we might see this with "Sr." or "III".

Reply:
I figured that since those cases are not present in our data so far, they 
are not likely to crop up in the future -- though I think adding them would be fine.
So far we have lots of "III" but they never appear with a comma.

Reply:
Okay, let's just deal with them as they arise.